doc/Moderation.md

```diff
@@ -105,7 +105,8 @@ Moderators are often the first to see and triage new issues, and so they have th
 * `Issue-Bug`
 * `Issue-Docs`
 * `Issue-Feature`
-* `Last-Version-Remaining`
+* `Highest-Version-Removal`
+* `Last-Version-Removal`
```

Comment:
Since Trenly pointed out the sort order in another file, it would also help if we could move `Highest-Version-Removal` to its lexicographical position as well :D